src/components/preview/__snapshots__/preview.spec.js.snap

```diff
@@ -0,0 +1,163 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Preview when given a width and height renders with the given width and height 1`] = `
+.c1 {
+  -webkit-animation: blNbyL 2s ease infinite;
+  animation: blNbyL 2s ease infinite;
```

Comment:
wouldn't that be different on each snapshot generation?

Reply:
It should works good